app/models/user.rb

```diff
@@ -44,4 +46,10 @@ def needs_to_see_privacy_notice?
   def seen_privacy_notice?
     privacy_notice_seen_at.present?
   end
+
+  def force_email_address_to_lowercase!
+    # the 'self' scope is required, for some reason -
```

Comment:
I think the self scope is required otherwise Ruby defines it as a local variable, rather than calling the setter method.